Sources/Widgets/Widgets3D/ResliceCursorWidget/index.js

```diff
@@ -0,0 +1,526 @@
+        bounds[0] + offset[0],
+        center[1],
+        bounds[4] + offset[2]
+      );
+      planeSource.setPoint1(
+        bounds[1] - offset[0],
+        center[1],
+        bounds[4] + offset[2]
+      );
+      planeSource.setPoint2(
+        bounds[0] + offset[0],
+        center[1],
+        bounds[5] - offset[2]
+      );
+    } else if (viewType === ViewTypes.AXIAL) {
+      planeSource.setOrigin(
+        bounds[0] + offset[0],
+        bounds[2] + offset[1],
+        center[2]
+      );
+      planeSource.setPoint1(
+        bounds[1] - offset[0],
+        bounds[2] + offset[1],
+        center[2]
+      );
+      planeSource.setPoint2(
+        bounds[0] + offset[0],
+        bounds[3] - offset[1],
+        center[2]
+      );
+    } else if (viewType === ViewTypes.SAGITTAL) {
+      planeSource.setOrigin(
+        center[0],
+        bounds[2] + offset[1],
+        bounds[4] + offset[2]
+      );
+      planeSource.setPoint1(
+        center[0],
+        bounds[3] - offset[1],
+        bounds[4] + offset[2]
+      );
+      planeSource.setPoint2(
+        center[0],
+        bounds[2] + offset[1],
+        bounds[5] - offset[2]
+      );
+    }
+    return planeSource;
+  }
+
+  function updateCamera(renderer, normal) {
+    // When the reslice plane is changed, update the camera to look at the
+    // normal to the reslice plane.
+
+    const focalPoint = renderer.getActiveCamera().getFocalPoint();
+
+    const distance = renderer.getActiveCamera().getDistance();
+
+    const estimatedCameraPosition = [
+      focalPoint[0] + distance * normal[0],
+      focalPoint[1] + distance * normal[1],
+      focalPoint[2] + distance * normal[2],
+    ];
+
+    // intersect with the plane to get updated focal point
+    const intersection = vtkPlane.intersectWithLine(
+      focalPoint,
+      estimatedCameraPosition,
+      model.widgetState.getCenter(),
+      normal
+    );
+    const newFocalPoint = intersection.x;
+
+    renderer
+      .getActiveCamera()
+      .setFocalPoint(newFocalPoint[0], newFocalPoint[1], newFocalPoint[2]);
+
+    const newCameraPosition = [
+      newFocalPoint[0] + distance * normal[0],
+      newFocalPoint[1] + distance * normal[1],
+      newFocalPoint[2] + distance * normal[2],
+    ];
+
+    renderer
+      .getActiveCamera()
+      .setPosition(
+        newCameraPosition[0],
+        newCameraPosition[1],
+        newCameraPosition[2]
+      );
+
+    // Renderer may not have yet actor bounds
+    const rendererBounds = renderer.computeVisiblePropBounds();
+    const bounds = model.widgetState.getImage().getBounds();
+    rendererBounds[0] = Math.min(bounds[0], rendererBounds[0]);
+    rendererBounds[1] = Math.max(bounds[1], rendererBounds[1]);
+    rendererBounds[2] = Math.min(bounds[2], rendererBounds[2]);
+    rendererBounds[3] = Math.max(bounds[3], rendererBounds[3]);
+    rendererBounds[4] = Math.min(bounds[4], rendererBounds[4]);
+    rendererBounds[5] = Math.max(bounds[5], rendererBounds[5]);
```

Comment:
use vtkBoundingBox or vtkBox for such work on bounds

Reply:
There is no method which doing that job in these classes. Is it relevant to adding a method which takes two bounds and return the maximum bounds ?

Reply:
`vtkBoundingBox` has a `addBounds` method that does what you want...